### DISCUSSION TITLE
Podcast Features

### DISCUSSION THREAD

**Discussion by oenu** [Stage-Whisper/Stage-Whisper#25] [Ideas]
Discussion of features/ideas around podcast producers and podcast listeners.

## Podcast Producers
- Generate ~~SRT~~ VTT files for video podcasts or for podcast players that support ~~SRT~~ VTT

## Podcast Listeners
- Allow RSS links to be used as input source
- RSS batch feed processing

**Comment by petersterne** [comment]
Do podcast players generally support SRT rather than VTT? What is VTT used for then?

**Reply by oenu** [reply]
You are correct it is VTT for video players, changing original post

**Comment by petersterne** [comment]
The latest version of whisper now produces both SRT and VTT files by default (along with TXT files with no timestamps). So we don't have to do anything extra there.

But taking RSS links as inputs is a potential feature to consider.
